### ISSUE TITLE
OnConfigurationCompleted returns invalid Zone

### ISSUE THREAD

**Issue by yurivangeffen** [AdColony/AdColony-Unity-SDK-3#35]
When configuring my game using a zone defined in the AdColony backend, I get back a non-empty list of zones. The single zone that is in there (I also requested one) has a null/"" identifier and is not Enabled. This only happens on Android, iOS works fine. They seem to be configured the same in the backed.

Shouldn't the identifier be non-null/non-empty? And why does a similarly defined zone work properly on iOS, but not on Android?
The initialisation code:
```
AdColony.AppOptions appOptions = new AdColony.AppOptions();
appOptions.AdOrientation = AdColony.AdOrientationType.AdColonyOrientationLandscape;

AdColony.Ads.Configure(settings.AppId, appOptions, settings.ZoneIds);
AdColony.Ads.OnConfigurationCompleted += (List<AdColony.Zone> zones) => 
{
    if (zones == null || zones.Count <= 0)
    {
        Debug.LogWarning("[AdColonyProvider] Unable to configure.");
        _isInitialized = false;
    } else {
        Debug.Log("[AdColonyProvider] Configured.");
        _isInitialized = true;

        RegisterForAdsCallbacks();

        // Find zones we can use
        AdColony.Zone zone = zones.Find((AdColony.Zone z) => z.Enabled);
        if(zone == null)
        {
            Debug.LogWarning("[AdColonyProvider] No relevant zones found!");
            return;
        }

        AdColony.Ads.RequestInterstitialAd(zone.Identifier, _adOptions);
    }
};
```

**Comment by yurivangeffen** [comment]
Even giving AdColony.Ads.RequestInterstitialAd(...) a hardcoded identifier gives me no ads back. What could be causing this? Is it an SDK issue or a backend issue?

**Comment by shawnkc** [comment]
This will be fixed in versions 3.3.5+.
